### PR TITLE
Wallet drop

### PR DESCRIPTION
**Pull request by monstrobishi**
- Added functionality to add wallet manually.

### CODE REVIEW

**Comment by monstrobishi** [comment]
@Sumit570 

while we're at it, can we rename "Wallets" in the side bar to "Balances", and change the + button to say "+ ADD BALANCE"

**Comment by monstrobishi** [comment]
Also the title at the top "Wallets" -> "Balances"

File: webapp/src/containers/WalletPage/components/Tokens/TokensList/index.tsx [hunk 1]
```diff
@@ -58,38 +79,108 @@ const WalletTokensList: React.FunctionComponent<WalletTokensListProps> = (
...
     const keys = {};
     clone.forEach((t) => {
       keys[t.hash] = t.symbol;
     });
     appTokens = (appTokens || []).filter((t) => !keys[t.hash]);
     clone = [...clone, ...appTokens]
       .sort((a: IToken, b: IToken) => +a.hash - +b.hash)
-      .filter((t: IToken) => t.hash != DFI_SYMBOL);
-    const tableData = clone.slice(
+      .filter((t: IToken) => t.hash != '0');
```
Comment (thedoublejay) [inline webapp/src/containers/WalletPage/components/Tokens/TokensList/index.tsx:116]: You can replace the '0' with DFI_SYMBOL

File: webapp/src/containers/WalletPage/service.tsx [hunk 2]
```diff
@@ -69,6 +76,39 @@ export const handelRemoveReceiveTxns = (id, networkName) => {
...
+  const networkType = getNetworkType();
+  const initialData = JSON.parse(PersistentStore.get('tokenInfo') || '{}');
+  const keyData = [...(initialData[networkType] || []), tokenData];
+  initialData[networkType] = keyData;
+  PersistentStore.set('tokenInfo', initialData);
+  return initialData[networkType];
+};
+
+export const getWalletToken = () => {
+  const networkType = getNetworkType();
```
Comment (thedoublejay) [inline webapp/src/containers/WalletPage/service.tsx:89]: I think we can create a common method for these two lines. It's been used for the other methods as well.

File: webapp/src/containers/WalletPage/components/Tokens/TokensList/index.tsx [hunk 3]
```diff
@@ -58,38 +79,108 @@ const WalletTokensList: React.FunctionComponent<WalletTokensListProps> = (
...
+      ...accountTokens,
+      ...walletTableData,
+      ...verifiedTokens,
+    ].map((value) => value.symbolKey);
+    let tokenDataArray: any[] = [];
+    tokenData.forEach((value, key) => {
+      tokenDataArray = [...tokenDataArray, value.symbolKey];
+    });
+    const diffreneceArray = difference(existingTokenArray, tokenData);
+    return diffreneceArray.length > 1 ? true : false;
```
Comment (thedoublejay) [inline webapp/src/containers/WalletPage/components/Tokens/TokensList/index.tsx:182]: No need for true/false here. diffreneceArray.length > 1 would return boolean result. Minor typo on diffreneceArray  -> difference

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by thedoublejay — DISMISSED** [review]

**Comment by monstrobishi** [comment]
@thedoublejay can you check if this is good to merge?

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**